src/lib/Event/ContentProxyCreateEvent.php

```diff
@@ -26,14 +26,19 @@ class ContentProxyCreateEvent extends Event
     /** @var int */
     protected $parentLocationId;
 
+    /** @var \EzSystems\EzPlatformAdminUi\Event\Options */
+    protected $options;
+
     public function __construct(
         ContentType $contentType,
         string $languageCode,
-        int $parentLocationId
+        int $parentLocationId,
+        Options $options = null
```

Comment:
Since we allow to not pass this parameter, shouldn't we also explicitly allow passing `null` value?